setup.cfg

```diff
@@ -76,6 +76,7 @@ install_requires =
     cookiecutter >= 1.6.0, != 1.7.1
     python-gilt >= 1.2.1, < 2
     Jinja2 >= 2.10.1
+    packaging
```

Comment:
better to use `from distutils.version import LooseVersion`
no need to add dependencies
It works in `podman_container` like:
```python
if minv and LooseVersion(minv) > LooseVersion(self.podman_version):
   ...
```
https://github.com/sshnaidm/ansible-podman-collections/blob/8bfe7402f02dea320d116278739180b5c5c85a55/plugins/modules/podman_container.py#L892-L893

Reply:
In fact I went to pkg_resources as is newer and already used.